cuebot/build.gradle

```diff
@@ -1,36 +1,23 @@
-
-import org.gradle.api.tasks.testing.logging.TestExceptionFormat
-import org.gradle.api.tasks.testing.logging.TestLogEvent
-
-buildscript {
-    repositories {
-        mavenCentral()
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.8'
-        classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.2.1.RELEASE'
-        classpath 'org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.7.1'
-    }
+plugins {
+    id 'java'
+    id('eclipse')
+    id('idea')
+    id('org.springframework.boot') version "2.2.1.RELEASE"
+    id('io.spring.dependency-management') version "1.1.4"
+    id('com.google.protobuf') version "0.9.1"
+    id('jacoco')
+    id('org.sonarqube') version "2.8"
 }
 
-apply plugin: 'java'
-apply plugin: 'eclipse'
-apply plugin: 'idea'
-apply plugin: 'org.springframework.boot'
-apply plugin: 'io.spring.dependency-management'
-apply plugin: 'com.google.protobuf'
-apply plugin: 'jacoco'
-apply plugin: 'org.sonarqube'
-
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 11
+targetCompatibility = 11
 
 ext {
     activemqVersion = '5.12.0'
```

Comment:
nit: Can this use `def` like grpc / protobuf use now?

Reply:
I guess so. I've updated on the last commit to give it a go.